swap_meet/decor.py

```diff
@@ -0,0 +1,13 @@
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, condition = 0):
+        self.category = "Decor"
+        self.condition = condition
```

Comment:
This function is repeating some of the work that's done in the Item constructor.  Consider using `super().__init__(...)` as a way to DRY up code in subclasses when possible.